### PR TITLE
html form and table

### PR DESCRIPTION
**Pull request by KalyaniPayyavula**
- Added table
- Added from

image

- added img, li in a table
image

### CODE REVIEW

File: AboutUs/aboutUs.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+<head>
+
+</head>
+
+<body>
+    <form>
+        <label for="username">User name</label>
+        <input id="username" type="text">
+    </form></br>
```
Comment (yugandhar-pathi) [inline AboutUs/aboutUs.html]: Remove extra closing `</form>`

File: AboutUs/aboutUs.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        <option>Usa</option>
+        <option>Russia</option>
+        <option>China</option>
+        <option>Dubai</option>
+    </select></br></br>
+    <label for="date_id">Date of birth</label>
+    <input type="date" id="date_id"></br></br>
+    <label for="marital_status">Marital Status</label>
+    <input type="checkbox" id="marital_status"></input></br></br>
+    <label id="gender_checkbox">Gender</label>
```
Comment (yugandhar-pathi) [inline AboutUs/aboutUs.html]: I think you need to use `for` attribute instead of `id`

File: AboutUs/aboutUs.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+function onFormSubmit() {
+    const username = document.getElementById("username").value
+    const password = document.getElementById("password").value
+    const date = document.getElementById("date_id").value
+    const checkbox = document.getElementById("marital_status").checked
+    const maleRadio = document.getElementById("male_radio").checked
+    const femaleRadio = document.getElementById("female_radio").checked
```
Comment (yugandhar-pathi) [inline AboutUs/aboutUs.js:8]: I think there should be a better way to find what radio option is selected.

File: Career/career.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+        <li><img src="../Images/20220321113532064.jpg" alt="Skandhana" height="170px"></img></li>
+        <li><b>Name : Skandhana</b></li>
+        <li><b>Grade : 1st Grade</b></li>
+    </ul>
+
+    <ul>
+        <li><img src="../Images/20220321113910381.jpg" alt="Kalyani" height="170px"></img></li>
+        <li><b>Name : Kalyani</b></li>
+        <li><b>Grade : Kinder Garden</b></li>
+    </ul>
```
Comment (yugandhar-pathi) [inline Career/career.html]: Have single list. Make each student an item in the list.

**Review by yugandhar-pathi — CHANGES_REQUESTED** [review]

File: AboutUs/aboutUs.html (hunk 1 above)
Comment (KalyaniPayyavula) [reply AboutUs/aboutUs.html]: done

File: AboutUs/aboutUs.html (hunk 2 above)
Comment (KalyaniPayyavula) [reply AboutUs/aboutUs.html]: done

File: AboutUs/aboutUs.js (hunk 3 above)
Comment (KalyaniPayyavula) [reply AboutUs/aboutUs.js:8]: done

File: Career/career.html (hunk 4 above)
Comment (KalyaniPayyavula) [reply Career/career.html]: done

**Review by yugandhar-pathi — APPROVED** [review]

**Pull request merged**